js/components/Toolbox.js

```diff
@@ -0,0 +1,28 @@
+const React = require("react");
+const { connect } = require("react-redux");
+const dom = React.DOM;
+
+require('./toolbox.css');
+const App = React.createFactory(require('./app'));
+const Tabs = React.createFactory(require('./tabs'));
+const { getSelectedTab } = require('../queries');
+
+
+function isEmpty(obj) {
+  return Object.keys(obj).length == 0;
+}
+
+function Toolbox({ selectedTab }) {
+  const container =  isEmpty(selectedTab) ? Tabs() : App();
```

Comment:
seamless-immutable objects are normal JS objects. It just freezes them and augments them with a few more methods. But they are totally normal JS objects.

I think we should bite the bullet and switch everything to immutable.js though. I've had some troubles with seamless-immutable: https://gist.github.com/jlongster/028c83ed09d99c09983a#seamless-immutable